Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1523,6 +1738,146 @@ def get_context_table_csv(client: Client, args: Dict[str, str]) -> Tuple[Any, Di
     return f'Successfully downloaded Context Table CSV file {context_table_name}.', {}, None
 
 
+def get_notable_assets(client: Client, args: Dict) -> Tuple[Any, Dict[str, Any], Optional[Any]]:
+    """  Updates records of a context table.
+
+    Args:
+        client: Client
+        args: Dict
+
+    """
+    limit: int = args.get('limit', 10)
+    time_period: str = args.get('time_period', '')
+    time_ = time_period.split(' ')
+    if not len(time_) == 2:
+        raise Exception('Got invalid time period. Enter the time period number and unit.')
+    num: str = time_[0]
+    unit: str = time_[1]
+    api_unit = unit[0]
+    if api_unit == 'm':
+        api_unit = api_unit.upper()
+
+    if api_unit not in {'d', 'y', 'M', 'h'}:
+        raise Exception('The time unit is incorrect - can be hours, days, months, years.')
+
+    notable_assets_raw_data = client.get_notable_assets_request(api_unit, num, limit)
+    notable_assets = notable_assets_raw_data.get('assets')
+
+    if not notable_assets:
+        return 'No users were found in this period of time.', {}, {}
+
+    contents: list = []
+    for asset in notable_assets:
+        asset_ = asset.get('asset', {})
+        highest_risk_sequence = asset.get('highestRiskSequence', {})
+        latest_asset_comment = asset.get('latestAssetComment', {})
+        contents = contents_append_notable_assets_info(contents, asset, asset_, highest_risk_sequence,
+                                                       latest_asset_comment)
+
+    entry_context = {'Exabeam.NotableAsset(val.ipAddress && val.ipAddress === obj.ipAddress)': contents}
+    human_readable = tableToMarkdown('Exabeam Notable Assets:', contents, removeNull=True)
+
+    return human_readable, entry_context, notable_assets
+
+
+def get_notable_session_details(client: Client, args: Dict[str, str]) -> Tuple[Any, Dict[str, Any], Optional[Any]]:
+    """  Updates records of a context table.
+
+    Args:
+        client: Client
+        args: Dict
+
+    """
+    asset_id = args.get('asset_id')
+    sort_by = args.get('sort_by')
+    sort_order = 1 if args.get('sort_order') == 'asc' else -1
+    limit = int(args['limit'])
+
+    session_details_raw_data = client.get_notable_session_details_request(asset_id, sort_by, sort_order, limit)
+
+    contents: list = []
+    users: list = []
+    executive_user_flags: list = []
+
+    for session in session_details_raw_data.get('sessions', {}):
+        contents = contents_append_notable_session_details(contents, session)
+
+    users_response = session_details_raw_data.get('users', {})
+    for user_name, user_details in users_response.items():
+        user_info = user_details.get('info', {})
+        users = contents_append_notable_session_user_details(users, user_details, user_info)
+
+    executive_user = session_details_raw_data.get('executiveUserFlags', {})
+    for username, status in executive_user.items():
+        executive_user_flags.append({
+            username: status
+        })
```

Comment:
what is this data? is it something relevant to the command?

Reply:
It returns from the API under the same level as - `users` and `sessions`
So it looks like something related to the command.

Reply:
Did this go through design? Do we understand what it means?